classes/app.rb

```diff
@@ -4,7 +4,8 @@
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'classroom'
-
+require_relative 'preserve_data'
+# rubocop:disable Metrics/ClassLength
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly do not disable any **rubocop** property. It help to follow the standard and best practices when coding. Therefore I suggest that you find a way to reduce the length of the class `App` instead of disabling the `Metrics/ClassLength` property
**HIT:** You can move some of the methods in the `App` class in a new module. You can read more about [module in ruby here](https://ruby-doc.org/core-2.5.1/Module.html)